java/vector/src/test/java/org/apache/arrow/vector/TestVarCharListVector.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.arrow.vector;
```

Comment:
This files needs an Apache license header.

Reply:
@xhochy yup missed it..added them now.